src/azure-cli/azure/cli/command_modules/appservice/tests/latest/test_webapp_commands.py

```diff
@@ -590,6 +590,10 @@ def test_linux_webapp(self, resource_group):
 class LinuxWebappSSHScenarioTest(ScenarioTest):
     @ResourceGroupPreparer(location='japanwest')
     def test_linux_webapp_ssh(self, resource_group):
+        # Skip this test on Windows as it will fail at 'webapp ssh'
+        import platform
+        if platform.system() == "Windows":
```

Comment:
As there are many people using Windows system, if we skip this test, the test may be missed in some scenarios. But if we can make sure our CI system always have tests on Linux system. I am fine to skip this test because our CI system is running on Linux now. +@haroldrandom for awareness.
In addition, I will prefer that when system detected is Windows, the test can assert exception as expected.

Reply:
Nice idea, but `webapp ssh` does more than failing on Windows directly - it generates some network traffic before the platform detection: 
https://github.com/Azure/azure-cli/blob/c5b5772ab958e755a9649e455a8e6bb17edc6400/src/azure-cli/azure/cli/command_modules/appservice/custom.py#L3276-L3284

I have changed the logic of `ssh_webapp` so that it fails as early as possible.

Reply:
I would prefer to use decorator `unittest.skipIf()` leverage this.

Reply:
It is tested on Windows as well for raising the exception.

Reply:
I am not sure what's the purpose exactly? Is going to raise on Windows or Skip if on Windows.
If you want to skip on Windows, I would recommend to use `unitest.skipIf()` decorator.

Reply:
The command itself will raise an exception on Windows and the exception is also tested. So we are not skipping anything.